src/Plugin/Action/Order/ExportOrderAction.php

```diff
@@ -31,7 +31,7 @@ public function __construct(AbstractPdkOrderRepository $orderRepository, Shipmen
      */
     public function handle(array $parameters): Response
     {
-        $orders = $this->orderRepository->getMany($parameters['orderIds']);
+        $orders = $this->orderRepository->getMany((array) $parameters['orderIds']);
```

Comment:
Dit werkt niet als er meerdere orderIds in zitten, los het dan meteen goed op door netjes het format `id;id;id` te splitten. Misschien is het zelfs nice als `getMany` dat zelf doet